src/test/java/com/checkmarx/flow/cucumber/integration/negative_tests/RunJiraNegativeSteps.java

```diff
@@ -6,8 +6,8 @@
 
 @RunWith(Cucumber.class)
 @CucumberOptions(plugin = { "pretty", "summary", "html:build/cucumber/integration/negative_tests", "json:build/cucumber/integration/negative_tests/cucumber.json" },
-        features = "src/test/resources/cucumber/features",
+        features = "src/test/resources/cucumber/features/integrationTests/jira/publish-processing.feature",
```

Comment:
Does it mean that other tests under src/test/resources/cucumber/features won't run as part of RunJiraNegativeSteps?
I see that you decreased the scope to only one specific feature file. Is this what you meant to do?